07-regression1.Rmd

```diff
@@ -202,7 +217,6 @@ now. Note that for the remainder of the chapter we'll be working with the
 entire Sacramento data set, as opposed to the smaller sample of 30 points above.
 
 ```{r 07-test-train-split}
-set.seed(1234)
 sacramento_split <- initial_split(sacramento, prop = 0.6, strata = price)
 sacramento_train <- training(sacramento_split)
 sacramento_test <- testing(sacramento_split)
```

Comment:
I think below we should take a sentence or two to explain why we cannot use accuracy in the regression setting (with predicting numerical values, almost none of our predicted values will exactly match the true value for that observation, so accuracy doesn't really work in this setting).

Reply:
Also, when we talk about RMSPE here, should we talk about why we use the squared difference instead of just the difference? Most regression texts would address this, as it's not obvious to new learners who have never seen this before... Maybe a figure like this that we use in our slides (https://ubc-dsci.github.io/dsci-100-assets/2021-spring/slides/08_regression1.slides.html#/9) would be good?

<img width="281" alt="Screen Shot 2021-07-13 at 11 20 56 PM" src="https://user-images.githubusercontent.com/6865016/125573046-ebcfc837-01b4-443c-a1c1-9905ad5ad3d9.png">

Reply:
In our note about RMSE vs RMSPE, or elsewhere in the book, we never really explain why you might want to calculate RMSE. What can it tell us? Or when is it useful. It's kind of confusing that we point this difference out without explaining one of the things.

Reply:
done all the above